pkg/http/api.go

```diff
@@ -55,7 +55,7 @@ const (
 	retryPatternParam   = "retryPattern"
 	retryThresholdParam = "retryThreshold"
 	concurrencyParam    = "concurrency"
```

Comment:
I think we decided to keep the old one for http headers.

Reply:
it is not used by header. https://github.com/dapr/dapr/blob/841d0e6221b6c4f824b392c5531f47df83fe23a0/pkg/http/api.go#L404

Reply:
@artursouza let me know if I am wrong :)

Reply:
Great!